### DISCUSSION TITLE
CameraView not found

### DISCUSSION THREAD

**Discussion by CodingOctocat** [CommunityToolkit/Maui#2002] [Q&A] [answered]
I noticed that the CommunityToolkit documentation has a full tutorial for CameraView, but why can't I use it?

[communitytoolkit-docs/CameraView](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/views/camera-view?tabs=android)

![image](https://github.com/CommunityToolkit/Maui/assets/7220248/5e8b5870-adaf-4a27-af60-104003750de7)

**Comment by bijington** [comment, marked as answer]
Have you added the NuGet package as suggested in the getting started guide: https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/get-started?tabs=CommunityToolkitMauiCamera

**Reply by CodingOctocat** [reply]
Thanks, I made a mistake, I always thought CommunityToolkit.Maui included CameraView.

**Comment by CliffAgius** [comment]
Closed as Answered.
